client/src/components/dashboard/Tabs/index.js

```diff
@@ -1,76 +1,135 @@
-          <img src={staticImg1} alt="dash" className="dash__bar__img" />
-        </Link>
-      </li>
-      <li>
-        <Link className="dashLink" to="/dashboard/applications/accepted">
-          <img
-            src={staticImg4}
-            alt="Accepted Applications"
-            className="dash__bar__img"
-          />
-        </Link>
-      </li>
-      <li>
-        <Link className="dashLink" to="/dashboard">
-          <img src={staticImg6} alt="dash" className="dash__bar__img" />
-        </Link>
-      </li>
-      <li>
-        <Link className="dashLink" to="/dashboard">
-          <img src={staticImg2} alt="dash" className="dash__bar__img" />
-        </Link>
-      </li>
-      <li>
-        <Link className="dashLink" to="/dashboard">
-          <img src={staticImg7} alt="dash" className="dash__bar__img" />
-        </Link>
-      </li>
-      <li>
-        <Link className="dashLink" onClick={logOut}>
-          <img src={staticImg5} alt="logout" className="dash__bar__img" />
-        </Link>
-      </li>
-    </ul>
-  </div>
-);
+const getImg = (url) => {
+  const { pathname } = window.location;
+  const path = pathname.split('/');
+  if (path[path.length - 1] === url) return true;
+  return false;
+};
+
+const DashBar = ({ logOut }) => {
+  const [imgLog, setImgLog] = useState(logoutIcon);
 
+  return (
+    <div className="dash__bar">
+      <img src={logo} alt="Logo" className="logo" />
+      <ul className="dash__bar__link">
+        <li>
+          <Link title="Home" className="dashLink" to="/dashboard">
+            <img
+              src={getImg('dashboard') ? homeImg : home}
+              alt="dash"
+              className="dash__bar__img"
+            />
+          </Link>
+        </li>
+        <li>
+          <Link
+            title="Opened Applications"
+            className="dashLink"
+            to="/dashboard/applications/opened"
+          >
+            <img
+              src={getImg('opened') ? toolsImg : tools}
+              alt="dash"
+              className="dash__bar__img"
+            />
+          </Link>
+        </li>
+        <li>
+          <Link
+            title="Submitted Applications"
+            className="dashLink"
+            to="/dashboard/applications/submitted"
+          >
+            <img
+              src={getImg('submitted') ? submittedImg : submitted}
+              alt="dash"
+              className="dash__bar__img"
+            />
+          </Link>
+        </li>
+        <li>
+          <Link
+            title="Completed Applications"
+            className="dashLink"
+            to="/dashboard/applications/completed"
+          >
+            <img
+              src={getImg('completed') ? completedImg : completed}
+              alt="dash"
+              className="dash__bar__img"
+            />
+          </Link>
+        </li>
+        <li>
+          <Link
+            title="ِAccepted Applications"
```

Comment:
remove sign before A letter
```suggestion
            title="Accepted Applications"
```